data_browser/src/ExperimentPage.js

```diff
@@ -28,17 +39,16 @@ function ExperimentPage() {
         const experiment = response.data.data;
         setExperiment(experiment);
 
+        // Pretch all the urls (e.g., for status, results)
+        const urls = getAllPretchUrls(experiment);
```

Comment:
is pretch short for prefetch?

Reply:
oops, this is how new words are invented

Reply:
i mean, it's not a bad word